blocks/cards/cards.css

```diff
@@ -72,10 +77,84 @@
     align-self: center;
   }
 
+  .block.cards.image-cards {
+    flex-wrap: nowrap !important;
+  }
+
   .block.cards .card > div:has(.title) {
     display: flex;
     align-self: stretch;
     align-items: center;
     gap: 10px;
   }
 }
+
+.block.cards.image-cards {
+  padding: 0;
+  position: relative;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block.cards.image-cards .card {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  cursor: pointer;
+}
+
+.block.cards.image-cards .card-img {
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+}
+
+.block.cards.image-cards .card-img img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  opacity: 0.9;
+  border-radius: 16px;
+  z-index: -1;
+}
+
+.block.cards.image-cards .title {
+  z-index: 1;
+  color: white;
+  margin: 0;
+  align-self: flex-start;
+  font-size: 1.2rem;
+}
+
+.block.cards.image-cards .card > div:has(.title) {
+  display: flex;
+}
+
+.block.cards.image-cards .url {
+  opacity: 1;
+  height: unset;
+  width: unset;
+  position: initial;
+  font-size: 0.75rem;
+  z-index: 1;
+}
+
+.block.cards.image-cards .url a {
+  background-color: #f26841;
```

Comment:
This is giving accessibility issue:
Please try to use code: `A42E0E` for background color to fix the color contrast issue